### DISCUSSION TITLE
NocoDB setup on Debian [May 02]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#2003] [Discord Threads]
> 🙋‍♂️ **AngelOfElements**

**Comment by o1lab** [comment]
> 👨‍🚀 **wingkwong**  
#### https://github.com/nocodb/nocodb#quick-try   
####    
#### u can take a look at this. apart from the first one, others should work fine on your VM with Debian  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/970033830209667163)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
